org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/IteratorExtensions.java

```diff
@@ -1043,4 +1062,27 @@ protected T computeNext() {
 		}
 		return max;
 	}
+
+    /**
+     * Returns <tt>true</tt> if this Iterator contains the specified element.
+     * More formally, returns <tt>true</tt> if and only if this Iterator contains
+     * at least one element <tt>e</tt> such that
+     * <tt>(o==null&nbsp;?&nbsp;e==null&nbsp;:&nbsp;o.equals(e))</tt>.
+     * 
+     * <p>Note that this will advance or even exhaust the given iterator.</p>
+     *
+     * @param iterable 
+     * 			the elements to test
+     * @param o 
+     * 			element whose presence in this Iterator is to be tested
+     * @return <tt>true</tt> if this Iterator contains the specified element
+     */
+	public static <T> boolean contains(Iterator<T> iterator, Object o) {
```

Comment:
Same remark about the method signature here.

Reply:
done